components/announcement_bar/cloud_trial_announcement_bar/index.ts

```diff
@@ -31,7 +31,7 @@ function mapStateToProps(state: GlobalState) {
     let isFreeTrial = false;
     let daysLeftOnTrial = 0;
 
-    if (isCloud && subscription && subscription.trial_end_at > 0 && subscription?.status === 'trialing') {
+    if (isCloud && subscription && subscription?.is_free_trial === 'true') {
```

Comment:
Why is `is_free_trial` a string?

Reply:
the pr from the server defines it as a string

Reply:
Yes, just curious why it can't be a boolean? @nickmisasi  This seems error prone

Reply:
I just also noticed that some config values from the server like EnableDeveloper are also handled in the same way. It is not ideal and potentially error prone, but since we have only two places where we are validating this now, I am not sure if it is worth modifying the value again.